chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1722,3 +1727,34 @@ impl Handler<PeerManagerMessageRequest> for PeerManagerActor {
         self.handle_peer_manager_message(msg, ctx)
     }
 }
+
+impl Handler<NetworkDebugStatus> for PeerManagerActor {
```

Comment:
Once you remove the dependency on client_primitives, rename "NetworkDebugStatus, DebugStatusResponse" to
"GetDebugStatus, DebugStatus":
- "Network" is implied by the package name
- the request name should be either a verb, or end with "Request"
- "Response" suffix is redundant as long as the response type can be interpreted on its own (i.e. without the request).